packages/react/src/components/UIShell/SideNav.tsx

```diff
@@ -107,6 +109,8 @@ function SideNavRenderFunction(
     [`${prefix}--side-nav--rail`]: isRail,
     [`${prefix}--side-nav--ux`]: isChildOfHeader,
     [`${prefix}--side-nav--hidden`]: !isPersistent,
+    [`${prefix}--cds--side-nav--expand-on-hover`]:
+      expanded || expandedViaHoverState,
```

Comment:
```suggestion
```

I think we can delete this, as it is toggling based on the same props as `${prefix}--side-nav--expanded`